app/validators/questionnaire_schema.py

```diff
@@ -209,6 +210,44 @@ def answers_with_context(self):
         self._answers_with_context = answers_dict
         return self._answers_with_context
 
+    @property
+    def lists_with_context(self):
+        if self._lists_with_context:
+            return self._lists_with_context
```

Comment:
Do we need this if/return, I don't think we would ever hit this at the moment?

Reply:
I went with the same approach we use in answers_with_context, since this is a property, we populate it only the initial time.

Reply:
I've added the change now after our discussion.